src/components/DetailsMetrics/DetailsMetrics.scss

```diff
@@ -174,29 +183,115 @@
       }
     }
 
-    &-invocation-content {
-      flex: 0 1 27%;
+    &-invocation-header {
       display: flex;
+      flex: 1;
+      justify-content: end;
+      align-items: baseline;
+      font-size: 14px;
+      font-weight: 500;
+      color: $primary;
+      height: 40px;
+
+      &_drift_icon_contrainer {
```

Comment:
What is the of multiple underscores. Underscores can be used only for modifiers (also related to other classes that were added)

Reply:
changed to _drift-icon-contrainer

Reply:
Underscores are used only for modifiers